### ISSUE TITLE
When shift+tabbing from toolbar back to website form button, NVDA should announce aria-labelledby

### ISSUE THREAD

**Issue by juliecheng** [nvaccess/nvda#13061]
### Steps to reproduce:

1. Open [https://codepen.io/dalecooper/full/zYdJeEx](https://codepen.io/dalecooper/full/zYdJeEx)
2. Tab to the `Yes` button
3. NVDA reads out the `aria-labelledby` (Is this a good codepen?) and button text (Yes)
3. Tab to the `No` button
4. NVDA reads out the button text (No)
4. Tab to navigate outside of the site and onto the toolbar
5. Shift + tab back to the `No` button
6. NVDA reads out the button text (No) but not the `aria-labelledby`
7. Maybe relevant - the 'No' button is outlined in red (browse mode) instead of blue (focus mode)

### Actual behavior:
`aria-labelledby` is not read out by NVDA when shift+tabbing back from a toolbar into a group containing a form button. the button is also in browse mode instead of focus mode

### Expected behavior:
`aria-labelledby` should be read out by NVDA when tabbing/shift+tabbing back into a group containing a button (form control). https://www.w3.org/TR/WCAG20-TECHS/ARIA17.html#ARIA17-description

NVDA works as expected if I force focus mode, and then shift+tab back to the button. Thank you!

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2021.2

#### Windows version:

Windows 10 Version 10.0.19043 Build 19043

#### Name and version of other software in use when reproducing the issue:

Chrome 95.0.4638.69 (Official Build)
Firefox 94.0.1
Microsoft Edge Version 95.0.1020.44 (Official build)

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

2021.1 - same behavior

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by ehollig** [comment]
Hello @juliecheng, I am not able to reproduce this issue using NVDA 2022.2.3. Tabbing to the yes button in both browse mode and focus mode from the top of the page, the aria-labeledby for the group is announced. Shift tabbing from the end of the page to the No button in both browse mode and focus mode, the aria-labeledby is appropriately announced.

If this is still an issue, please provide additional information so that we can look into this further.
